_shared/project/db.py

```diff
@@ -0,0 +1,62 @@
+import alembic.command
```

Comment:
This `db.py` file works for new projects and provides the SQLAlchemy session etc. Existing projects that already have their own stuff are going to add this file to their ignore lists, at least for now. (The stuff in this file is SQLAlchemy 2-only anyway so existing projects wouldn't even be able to use it yet.)